app/webpack/packs/application.js

```diff
@@ -11,6 +11,7 @@ import Vue from 'vue/dist/vue.esm';
 import TurbolinksAdapter from 'vue-turbolinks';
 import SignIn from '../components/authentication/sign_in.vue';
 import Profile from '../components/user_profile/profile.vue';
+import CookieLaw from '../components/cookie_law/cookie_law.vue';
```

Comment:
Is it possible to import and use the component from `vue-cookie-law` here directly, instead of wrapping it in another component?

Reply:
If we don't include our own component we can't use a custom theme so unfortunately no

Reply:
OK, fair enough.